tests/test_dvc.py

```diff
@@ -220,12 +221,13 @@ def test_get_exp_name_duplicate(tmp_dir, mocked_dvc_repo, mocker, caplog):
     assert msg in caplog.text
 
 
-def test_test_mode(tmp_dir, monkeypatch, mocked_dvc_repo):
+@pytest.mark.parametrize("dvcyaml_path", ["dvc.yaml", Path("dvcyaml/dvc.yaml")])
```

Comment:
Sorry for not getting back to you on this earlier, but I would vote to put this in a separate test in `tests/test_make_dvcyaml.py`. It's probably enough to add a parameter [here](https://github.com/iterative/dvclive/blob/68884628e5845f95b89c950016d1b9a1a190cb60/tests/test_make_dvcyaml.py#L425).

Reply:
Yeah, this makes sense.